lib/backend/k8s/k8s_test.go

```diff
@@ -1967,6 +1987,154 @@ var _ = testutils.E2eDatastoreDescribe("Test Watch support", testutils.Datastore
 		ctx = context.Background()
 	})
 
+	It("Should support watching Profiles", func() {
+		By("Watching a single Profile", func() {
+			name := "default"
+			watch, err := c.Watch(ctx, model.ResourceListOptions{Name: name, Kind: apiv3.KindProfile}, "")
+			Expect(err).NotTo(HaveOccurred())
+			event := ExpectAddedEvent(watch.ResultChan())
+			Expect(event.New.Key.String()).To(Equal("Profile(kns.default)"))
+		})
+		By("Watching all Profiles", func() {
+			watch, err := c.Watch(ctx, model.ResourceListOptions{Kind: apiv3.KindProfile}, "")
+			Expect(err).NotTo(HaveOccurred())
+			ExpectAddedEvent(watch.ResultChan())
+		})
+	})
+
+	It("Should support watching NetworkPolicies", func() {
+		createTestNetworkPolicy := func() {
+			np := networkingv1.NetworkPolicy{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "test-watch-net-policy",
+				},
+			}
+			res := c.ClientSet.NetworkingV1().RESTClient().Post().Resource("networkpolicies").Namespace("default").Body(&np).Do()
+			Expect(res.Error()).NotTo(HaveOccurred())
+		}
+		deleteAllNetworkPolicies := func() {
+			err := c.Clean()
+			Expect(err).NotTo(HaveOccurred())
+			nps := networkingv1.NetworkPolicyList{}
+			err = c.ClientSet.NetworkingV1().RESTClient().Get().Resource("networkpolicies").Timeout(10 * time.Second).Do().Into(&nps)
```

Comment:
Shouldn't need to do the call to `RESTClient()` here.

I think `NetworkingV1().NetworkPolicies().List(metav1.ListOptions{})` should do the trick (it does all the restclient stuff for you).

Reply:
Same for the other calls through to `RESTClient`